packages/tools/guiEditor/src/diagram/workbench.tsx

```diff
@@ -321,33 +321,116 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
         this.props.globalState.onPointerUpObservable.notifyObservers(null);
     };
 
+    /**
+     * Adds editor behavior to control
+     * @param guiControl
+     */
+    addEditorBehavior(guiControl: Control) {
+        const onPointerUpStored = guiControl.onPointerUpObservable.observers;
+        guiControl.onPointerUpObservable.clear();

```

Comment:
This takes an assumption that when you call clear you create a new array. otherwise the clear call will also clear the array you are storing.
To stay safe (just in case someone changes the `clear` function of the observable, make sure to slice the array (i.e. - make a shallow copy) instead of just referencing it.

Reply:
Same goes to the rest of the references :-)

Reply:
Ok done!